### PR TITLE
Added test cases for add and remove authors

### PR DESCRIPTION
**Pull request by rohibhan**
Test and Mock Setup for Authorship Pallet.

### CODE REVIEW

File: pallets/authorship/src/mock.rs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+			NodeBlock = Block,
+			UncheckedExtrinsic = UncheckedExtrinsic,
+		{
+			System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+			Authorship: pallet_authorship::{Pallet, Storage, Call, Event<T>},
+		}
+	);
+
+	parameter_types! {
+		pub const SS58Prefix: u8 = 38;
```
Comment (smohan-dw) [inline pallets/authorship/src/mock.rs]: Cord Prefix is 29

File: pallets/authorship/src/tests.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+use frame_support::{assert_noop, assert_ok, BoundedVec};
+
+use cord_utilities::mock::mock_origin;
+use frame_system::RawOrigin;
+use sp_runtime::DispatchError;
+
+use crate::{mock::*, Banned, DidNameOwnershipOf, Error, Names, Owner, Pallet};
```
Comment (smohan-dw) [inline pallets/authorship/src/tests.rs]: review these

Comment (rohibhan) [reply pallets/authorship/src/tests.rs]: Thanks. Got it. Will work on it and fix  issues asap.

File: pallets/authorship/src/tests.rs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+#[test]
+fn add() {
+	ExtBuilder::default().build().execute_with(|| {
+		let test_mock_authors: Vec<CordAccountOf<Test>> = vec![TEST_AUTHOR2,TEST_AUTHOR3];
+		assert_ok!(Authorship::add(
+			RawOrigin::Root.into(),
+			test_mock_authors.clone()
+		));
```
Comment (smohan-dw) [inline pallets/authorship/src/tests.rs:13]: This call should have failed. We need to add an error condition to the lib, to check for duplicates

**Comment by smohan-dw** [comment]
LGTM

**Review by smohan-dw — APPROVED** [review]
LGTM

**Pull request merged**